signup/index.php

```diff
@@ -49,6 +49,16 @@
       <br /> <br/>
       <a href="https://anmeldung.jugendhackt.org/jh-lab-isenbuettel/" class="btn btn-lg btn-secondary fw-bold border-white bg-white">All Sign-Ups / Alle Anmeldungen</a>
     </p>
+
+    <?php
+ $exists = file_exists('../online/'.$workshopslug.'.html'); 
```

Comment:
Here we're checking if the file ../online/$workshopslug.html exists. If yes, we're going to deliver it as an iframe, if not, we're gonna deliver the nocontent.html file as iframe to be consistent in the setup. (And I'm incredibly lazy.)